### ISSUE TITLE
Too many arguments reported in incorrect lines

### ISSUE THREAD

**Issue by Draqun** [PyCQA/pylint#5899]
### Bug description

When I called pylint there are reports about too many arguments in incorrect places of my tests. Source code is below.

Tests:

```python
import logging
from os.path import join
from unittest.mock import Mock, call, mock_open, patch

import pytest

from business_logic.azure import Azure


class TestAzure:
    @staticmethod
    @pytest.fixture
    def temp_dir():
        return Mock()

    @staticmethod
    @pytest.fixture
    def azure(temp_dir, app_config):
        with patch('business_logic.azure.TemporaryDirectory', temp_dir):
            yield Azure(app_config)

    @staticmethod
    def test_azure(temp_dir, app_config, caplog):
        with patch('business_logic.azure.TemporaryDirectory', temp_dir), caplog.at_level(logging.INFO):
            azure = Azure(app_config)

        assert temp_dir.call_count == 1
        assert temp_dir.call_args_list == [call()]
        assert caplog.messages == [f'Creating directory \'{temp_dir.return_value.name}\'.']
        assert hasattr(azure, 'datalake_api_key')
        assert hasattr(azure, 'datalake_storage_name')
        assert hasattr(azure, 'datalake_file_system')
        assert hasattr(azure, 'file_directory')

    @staticmethod
    @patch('builtins.open', new_callable=mock_open)
    @patch('business_logic.azure.mkdir')
    @patch('business_logic.azure.exists')
    @patch('business_logic.azure.DataLakeServiceClient')
    @pytest.mark.parametrize('exists_return_value', [True, False])
    @pytest.mark.parametrize('paths, output_paths, expected_result, messages', [
        (
            ['Xledger/test/file.txt'],
            ['/remote'],
            (['/remote/file.txt'], '/remote'),
            [
                'Preparing datalake service client.',
                'Start downloading files.',
                'Downloading \'Xledger/test/file.txt\' to \'/remote\'.',
                'Open \'/remote/file.txt\'.',
                'All files downloaded.',
            ],
        ),
        (
            ['Xledger/test-dir/'],
            ['/remote/test-dir'],
            (['/remote/test-dir/0.txt'], '/remote'),
            [
                'Preparing datalake service client.',
                'Start downloading files.',
                'Downloading \'Xledger/test-dir/0.txt\' to \'/remote/test-dir\'.',
                'Open \'/remote/test-dir/0.txt\'.',
                'All files downloaded.',
            ],
        ),
        (
            ['Xledger/test/file.txt', 'Xledger/test'],
            ['/remote', '/remote/test'],
            (['/remote/file.txt', '/remote/test/1.txt'], '/remote'),
            [
                'Preparing datalake service client.',
                'Start downloading files.',
                'Downloading \'Xledger/test/file.txt\' to \'/remote\'.',
                'Open \'/remote/file.txt\'.',
                'Downloading \'Xledger/test/1.txt\' to \'/remote/test\'.',
                'Open \'/remote/test/1.txt\'.',
                'All files downloaded.',
            ],
        ),
    ])
    def test_azure_download_files(  # pylint:disable=too-many-locals
            client_class_mock,
            exists_mock,
            mkdir_mock,
            open_mock,
            paths,
            output_paths,
            expected_result,
            messages,
            exists_return_value,
            azure,
            temp_dir,
            app_config,
            caplog,
    ):
        temp_dir.return_value.name = '/remote'
        # Mock(name='val') has special meaning and cannot be built in comprehensions
        remote_paths = [[Mock()] for _ in paths]
        for num, (path, mock_list) in enumerate(zip(paths, remote_paths)):
            mock_list[0].name = path if path.endswith('.txt') else join(path, f'{num}.txt')

        exists_mock.return_value = exists_return_value
        service_client = client_class_mock.return_value
        file_system_client = service_client.get_file_system_client.return_value
        file_system_client.get_paths.side_effect = remote_paths
        file_client = file_system_client.get_file_client.return_value
        file_stream = file_client.download_file.return_value
        file_stream.readall.return_value = b'TestData'

        with caplog.at_level(logging.INFO):
            result = azure.download_files(paths)

        assert client_class_mock.call_count == 1
        assert client_class_mock.call_args_list == [call(
            account_url=f'https://{app_config.datalake_storage_name}.dfs.core.windows.net',
            credential=app_config.datalake_api_key
        )]
        assert service_client.get_file_system_client.call_count == 1
        assert service_client.get_file_system_client.call_args_list == [
            call(file_system=app_config.datalake_file_system)
        ]
        assert file_system_client.get_paths.call_count == len(paths)
        assert file_system_client.get_paths.call_args_list == [call(path=path) for path in paths]

        assert file_system_client.get_file_client.call_count == len(paths)
        assert file_system_client.get_file_client.call_args_list == [call(path[0].name) for path in remote_paths]

        assert exists_mock.call_count == len(output_paths)
        assert exists_mock.call_args_list == [call(path) for path in output_paths]

        assert mkdir_mock.call_count == (len(output_paths) if not exists_return_value else 0)
        assert mkdir_mock.call_args_list == ([call(path) for path in output_paths] if not exists_return_value else [])

        assert open_mock.call_count == len(expected_result[0])
        assert open_mock.call_args_list == [call(path, 'wb') for path in expected_result[0]]

        assert result == expected_result
        assert caplog.messages == messages

```

```python
import logging
from os.path import join
from unittest.mock import Mock, call, patch

from freezegun import freeze_time
import pytest

from business_logic.archiver import Archiver


class TestArchiver:
    @staticmethod
    @pytest.fixture
    def temp_dir():
        return Mock()

    @staticmethod
    @pytest.fixture
    def archiver(temp_dir):
        with patch('business_logic.archiver.TemporaryDirectory', temp_dir):
            yield Archiver('test_password')

    @staticmethod
    def test_archiver(temp_dir, caplog):
        with patch('business_logic.archiver.TemporaryDirectory', temp_dir), caplog.at_level(logging.INFO):
            archiver = Archiver('another_test_password')

        assert temp_dir.call_count == 1
        assert temp_dir.call_args_list == [call()]
        assert caplog.messages == [f'Creating directory \'{temp_dir.return_value.name}\'.']
        assert hasattr(archiver, 'file_directory')

    @staticmethod
    @freeze_time('2022-02-02 2:22:22')
    @patch('business_logic.archiver.chdir')
    @patch('business_logic.archiver.ZipFile')
    @pytest.mark.parametrize('files, directory, messages', [
        (
            [],
            '/download',
            [
                'Start creating archive.',
                'Open zip file: \'/download/Files.2022-02-02_02.22.22.zip\'.',
                'Set password on zip file.',
            ],
        ),
        (
            ['/media/t1.crc'],
            '/media',
            [
                'Start creating archive.',
                'Open zip file: \'/media/Files.2022-02-02_02.22.22.zip\'.',
                'Archiving file: \'/media/t1.crc\'.',
                'Set password on zip file.',
            ],
        ),
        (
            ['/media/t1.crc', '/incorrect/path/src.txt'],
            '/media',
            [
                'Start creating archive.',
                'Open zip file: \'/media/Files.2022-02-02_02.22.22.zip\'.',
                'Archiving file: \'/media/t1.crc\'.',
                'Set password on zip file.',
            ],
        ),
        (
            ['/media/t1.crc', '/media/src.txt'],
            '/media',
            [
                'Start creating archive.',
                'Open zip file: \'/media/Files.2022-02-02_02.22.22.zip\'.',
                'Archiving file: \'/media/t1.crc\'.',
                'Archiving file: \'/media/src.txt\'.',
                'Set password on zip file.',
            ],
        ),
    ])
    def test_create_archives(zip_file_class, chdir_mock, files, directory, messages, archiver, temp_dir, caplog):  # pylint:disable=too-many-arguments
        temp_dir.return_value.name = directory
        zip_file = zip_file_class.return_value.__enter__.return_value
        zip_path = join(directory, 'Files.2022-02-02_02.22.22.zip')
        with caplog.at_level(logging.INFO):
            result = archiver.create_archives(files, directory)

        assert chdir_mock.call_count == 1
        assert chdir_mock.call_args_list == [call(directory)]

        assert zip_file_class.call_count == 1
        assert zip_file_class.call_args_list == [call(zip_path, 'w')]

        assert zip_file.write.call_count == len([file for file in files if file.startswith(directory)])
        assert zip_file.setpassword.call_count == 1
        assert zip_file.setpassword.call_args_list == [call('test_password'.encode('utf-8'))]
        assert result == zip_path
        assert caplog.messages == messages
```

### Configuration

```ini
[MASTER]

# A comma-separated list of package or module names from where C extensions may
# be loaded. Extensions are loading into the active Python interpreter and may
# run arbitrary code.
extension-pkg-whitelist=

# Add files or directories to the blacklist. They should be base names, not
# paths.
ignore=CVS

# Add files or directories matching the regex patterns to the blacklist. The
# regex matches against base names, not paths.
ignore-patterns=

# Python code to execute, usually for sys.path manipulation such as
# pygtk.require().
#init-hook=

# Use multiple processes to speed up Pylint. Specifying 0 will auto-detect the
# number of processors available to use.
jobs=0

# Control the amount of potential inferred values when inferring a single
# object. This can help the performance when dealing with large functions or
# complex, nested conditions.
limit-inference-results=100

# List of plugins (as comma separated values of python modules names) to load,
# usually to register additional checkers.
load-plugins=pylint.extensions.emptystring,
             pylint.extensions.bad_builtin,
             pylint.extensions.mccabe,
             pylint.extensions.check_elif,
             pylint.extensions.redefined_variable_type,
             pylint.extensions.overlapping_exceptions

# Pickle collected data for later comparisons.
persistent=yes

# Specify a configuration file.
#rcfile=

# When enabled, pylint would attempt to guess common misconfiguration and emit
# user-friendly hints instead of false-positive error messages.
suggestion-mode=yes

# Allow loading of arbitrary C extensions. Extensions are imported into the
# active Python interpreter and may run arbitrary code.
unsafe-load-any-extension=no


[MESSAGES CONTROL]

# Only show warnings with the listed confidence levels. Leave empty to show
# all. Valid levels: HIGH, INFERENCE, INFERENCE_FAILURE, UNDEFINED.
confidence=

# Disable the message, report, category or checker with the given id(s). You
# can either give multiple identifiers separated by comma (,) or put this
# option multiple times (only on the command line, not in the configuration
# file where it should appear only once). You can also use "--disable=all" to
# disable everything first and then reenable specific checks. For example, if
# you want to run only the similarities checker, you can use "--disable=all
# --enable=similarities". If you want to run only the classes checker, but have
# no Warning level messages displayed, use "--disable=all --enable=classes
# --disable=W".
disable=fixme,
        logging-too-many-args,
        logging-fstring-interpolation,
        missing-docstring,
        no-else-return,
        too-few-public-methods,
        redefined-outer-name,
        import-outside-toplevel,
        protected-access,

# Enable the message, report, category or checker with the given id(s). You can
# either give multiple identifier separated by comma (,) or put this option
# multiple time (only on the command line, not in the configuration file where
# it should appear only once). See also the "--disable" option for examples.
enable=c-extension-no-member


[REPORTS]

# Python expression which should return a note less than 10 (10 is the highest
# note). You have access to the variables errors warning, statement which
# respectively contain the number of errors / warnings messages and the total
# number of statements analyzed. This is used by the global evaluation report
# (RP0004).
evaluation=10.0 - ((float(5 * error + warning + refactor + convention) / statement) * 10)

# Template used to display messages. This is a python new-style format string
# used to format the message information. See doc for all details.
#msg-template=

# Set the output format. Available formats are text, parseable, colorized, json
# and msvs (visual studio). You can also give a reporter class, e.g.
# mypackage.mymodule.MyReporterClass.
output-format=text

# Tells whether to display a full report or only the messages.
reports=no

# Activate the evaluation score.
score=yes


[REFACTORING]

# Maximum number of nested blocks for function / method body
max-nested-blocks=5

# Complete name of functions that never returns. When checking for
# inconsistent-return-statements if a never returning function is called then
# it will be considered as an explicit return statement and no message will be
# printed.
never-returning-functions=sys.exit


[VARIABLES]

# List of additional names supposed to be defined in builtins. Remember that
# you should avoid defining new builtins when possible.
additional-builtins=

# Tells whether unused global variables should be treated as a violation.
allow-global-unused-variables=yes

# List of strings which can identify a callback function by name. A callback
# name must start or end with one of those strings.
callbacks=cb_,
          _cb

# A regular expression matching the name of dummy variables (i.e. expected to
# not be used).
dummy-variables-rgx=_+$|(_[a-zA-Z0-9_]*[a-zA-Z0-9]+?$)|dummy|^ignored_|^unused_

# Argument names that match this expression will be ignored. Default to name
# with leading underscore.
ignored-argument-names=_.*|^ignored_|^unused_

# Tells whether we should check for unused import in __init__ files.
init-import=no

# List of qualified module names which can have objects that can redefine
# builtins.
redefining-builtins-modules=six.moves,past.builtins,future.builtins,builtins,io

[STRING]

# This flag controls whether the implicit-str-concat-in-sequence should
# generate a warning on implicit string concatenation in sequences defined over
# several lines.
check-str-concat-over-line-jumps=no

# Replaces pylint-quotes, which is mostly dead and stopped working with pylint 2.8.2
check-quote-consistency=yes

[BASIC]

# Naming style matching correct argument names.
argument-naming-style=snake_case

# Regular expression matching correct argument names. Overrides argument-
# naming-style.
#argument-rgx=

# Naming style matching correct attribute names.
attr-naming-style=snake_case

# Regular expression matching correct attribute names. Overrides attr-naming-
# style.
#attr-rgx=

# Bad variable names which should always be refused, separated by a comma.
bad-names=foo,
          bar,
          baz,
          toto,
          tutu,
          tata

# Naming style matching correct class attribute names.
class-attribute-naming-style=snake_case

# Regular expression matching correct class attribute names. Overrides class-
# attribute-naming-style.
#class-attribute-rgx=

# Naming style matching correct class names.
class-naming-style=PascalCase

# Regular expression matching correct class names. Overrides class-naming-
# style.
#class-rgx=

# Naming style matching correct constant names.
const-naming-style=UPPER_CASE

# Regular expression matching correct constant names. Overrides const-naming-
# style.
#const-rgx=

# Minimum line length for functions/classes that require docstrings, shorter
# ones are exempt.
docstring-min-length=-1

# Naming style matching correct function names.
function-naming-style=snake_case

# Regular expression matching correct function names. Overrides function-
# naming-style.
#function-rgx=

# Good variable names which should always be accepted, separated by a comma.
good-names=i,
           j,
           k,
           ex,
           Run,
           _

# Include a hint for the correct naming format with invalid-name.
include-naming-hint=yes

# Naming style matching correct inline iteration names.
inlinevar-naming-style=any

# Regular expression matching correct inline iteration names. Overrides
# inlinevar-naming-style.
#inlinevar-rgx=

# Naming style matching correct method names.
method-naming-style=snake_case

# Regular expression matching correct method names. Overrides method-naming-
# style.
#method-rgx=

# Naming style matching correct module names.
module-naming-style=snake_case

# Regular expression matching correct module names. Overrides module-naming-
# style.
#module-rgx=

# Colon-delimited sets of names that determine each other's naming style when
# the name regexes allow several styles.
name-group=

# Regular expression which should only match function or class names that do
# not require a docstring.
no-docstring-rgx=^_

# List of decorators that produce properties, such as abc.abstractproperty. Add
# to this list to register other decorators that produce valid properties.
# These decorators are taken in consideration only for invalid-name.
property-classes=abc.abstractproperty

# Naming style matching correct variable names.
variable-naming-style=snake_case

# Regular expression matching correct variable names. Overrides variable-
# naming-style.
#variable-rgx=


[SPELLING]

# Limits count of emitted suggestions for spelling mistakes.
max-spelling-suggestions=4

# Spelling dictionary name. Available dictionaries: none. To make it working
# install python-enchant package..
spelling-dict=

# List of comma separated words that should not be checked.
spelling-ignore-words=

# A path to a file that contains private dictionary; one word per line.
spelling-private-dict-file=

# Tells whether to store unknown words to indicated private dictionary in
# --spelling-private-dict-file option instead of raising a message.
spelling-store-unknown-words=no


[TYPECHECK]

# List of decorators that produce context managers, such as
# contextlib.contextmanager. Add to this list to register other decorators that
# produce valid context managers.
contextmanager-decorators=contextlib.contextmanager

# List of members which are set dynamically and missed by pylint inference
# system, and so shouldn't trigger E1101 when accessed. Python regular
# expressions are accepted.
generated-members=

# Tells whether missing members accessed in mixin class should be ignored. A
# mixin class is detected if its name ends with "mixin" (case insensitive).
ignore-mixin-members=yes

# Tells whether to warn about missing members when the owner of the attribute
# is inferred to be None.
ignore-none=yes

# This flag controls whether pylint should warn about no-member and similar
# checks whenever an opaque object is returned when inferring. The inference
# can return multiple potential results while evaluating a Python object, but
# some branches might not be evaluated, which results in partial inference. In
# that case, it might be useful to still emit no-member and other checks for
# the rest of the inferred objects.
ignore-on-opaque-inference=yes

# List of class names for which member attributes should not be checked (useful
# for classes with dynamically set attributes). This supports the use of
# qualified names.
ignored-classes=optparse.Values,thread._local,_thread._local

# List of module names for which member attributes should not be checked
# (useful for modules/projects where namespaces are manipulated during runtime
# and thus existing member attributes cannot be deduced by static analysis. It
# supports qualified module names, as well as Unix pattern matching.
ignored-modules=

# Show a hint with possible names when a member name was not found. The aspect
# of finding the hint is based on edit distance.
missing-member-hint=yes

# The minimum edit distance a name should have in order to be considered a
# similar match for a missing member name.
missing-member-hint-distance=1

# The total number of similar names that should be taken in consideration when
# showing a hint for a missing member.
missing-member-max-choices=1


[FORMAT]

# Expected format of line ending, e.g. empty (any line ending), LF or CRLF.
expected-line-ending-format=

# Regexp for a line that is allowed to be longer than the limit.
ignore-long-lines=^\s*(# )?<?https?://\S+>?$

# Number of spaces of indent required inside a hanging or continued line.
indent-after-paren=4

# String used as indentation unit. This is usually "    " (4 spaces) or "\t" (1
# tab).
indent-string='    '

# Maximum number of characters on a single line.
max-line-length=120

# Maximum number of lines in a module.
max-module-lines=1000

# List of optional constructs for which whitespace checking is disabled. `dict-
# separator` is used to allow tabulation in dicts, etc.: {1  : 1,\n222: 2}.
# `trailing-comma` allows a space between comma and closing bracket: (a, ).
# `empty-line` allows space-only lines.
no-space-check=trailing-comma,
               dict-separator

# Allow the body of a class to be on the same line as the declaration if body
# contains single statement.
single-line-class-stmt=no

# Allow the body of an if to be on the same line as the test if there is no
# else.
single-line-if-stmt=no


[LOGGING]

# Format style used to check logging format string. `old` means using %
# formatting, while `new` is for `{}` formatting.
logging-format-style=new

# Logging modules to check that the string format arguments are in logging
# function parameter format.
logging-modules=logging


[MISCELLANEOUS]

# List of note tags to take in consideration, separated by a comma.
notes=FIXME,
      XXX,
      TODO


[SIMILARITIES]

# Ignore comments when computing similarities.
ignore-comments=yes

# Ignore docstrings when computing similarities.
ignore-docstrings=yes

# Ignore imports when computing similarities.
ignore-imports=no

# Minimum lines number of a similarity.
min-similarity-lines=10


[IMPORTS]

# Allow wildcard imports from modules that define __all__.
allow-wildcard-with-all=no

# Analyse import fallback blocks. This can be used to support both Python 2 and
# 3 compatible code, which means that the block might have code that exists
# only in one or another interpreter, leading to false positives when analysed.
analyse-fallback-blocks=no

# Deprecated modules which should not be used, separated by a comma.
deprecated-modules=optparse,tkinter.tix

# Create a graph of external dependencies in the given file (report RP0402 must
# not be disabled).
ext-import-graph=

# Create a graph of every (i.e. internal and external) dependencies in the
# given file (report RP0402 must not be disabled).
import-graph=

# Create a graph of internal dependencies in the given file (report RP0402 must
# not be disabled).
int-import-graph=

# Force import order to recognize a module as part of the standard
# compatibility libraries.
known-standard-library=

# Force import order to recognize a module as part of a third party library.
known-third-party=enchant


[DESIGN]

# Maximum number of arguments for function / method.
max-args=7

# Maximum number of attributes for a class (see R0902).
max-attributes=7

# Maximum number of boolean expressions in an if statement.
max-bool-expr=5

# Maximum number of branch for function / method body.
max-branches=12

# Maximum number of locals for function / method body.
max-locals=15

# Maximum number of parents for a class (see R0901).
max-parents=7

# Maximum number of public methods for a class (see R0904).
max-public-methods=20

# Maximum number of return / yield for function / method body.
max-returns=6

# Maximum number of statements in function / method body.
max-statements=50

# Minimum number of public methods for a class (see R0903).
min-public-methods=2


[CLASSES]

# List of method names used to declare (i.e. assign) instance attributes.
defining-attr-methods=__init__,
                      __new__,
                      setUp

# List of member names, which should be excluded from the protected access
# warning.
exclude-protected=_asdict,
                  _fields,
                  _replace,
                  _source,
                  _make

# List of valid names for the first argument in a class method.
valid-classmethod-first-arg=cls

# List of valid names for the first argument in a metaclass class method.
valid-metaclass-classmethod-first-arg=cls


[EXCEPTIONS]

# Exceptions that will emit a warning when being caught. Defaults to
# "Exception".
overgeneral-exceptions=Exception
```

### Command used

```shell
pylint --rcfile=scripts/config/pylintrc_tests tests/
```

### Pylint output

```shell
PYTHONPATH=":/home/damian/PycharmProjects/Mailer/src" pylint --rcfile=scripts/config/pylintrc_tests tests
************* Module tests.unit.business_logic.test_azure
tests/unit/business_logic/test_azure.py:78:4: R0913: Too many arguments (13/7) (too-many-arguments)
************* Module tests.unit.business_logic.test_archiver
tests/unit/business_logic/test_archiver.py:76:4: R0913: Too many arguments (8/7) (too-many-arguments)
```

### Expected behavior

Report bug in correct places
- tests/unit/business_logic/test_azure.py:81:4 instead tests/unit/business_logic/test_azure.py:78:4
- tests/unit/business_logic/test_archiver.py:79:4 instea tests/unit/business_logic/test_archiver.py:76:4

### Pylint version

```shell
pylint 2.12.2
astroid 2.9.3
Python 3.9.5 (default, Nov 23 2021, 15:27:38)
```

### OS / Environment

Ubuntu 20.04 LTS

### Additional dependencies

astroid==2.9.3; python_full_version >= "3.6.2"
atomicwrites==1.4.0; python_version >= "3.6" and python_full_version < "3.0.0" and sys_platform == "win32" or sys_platform == "win32" and python_version >= "3.6" and python_full_version >= "3.4.0"
attrs==21.4.0; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "3.6"
azure-core==1.23.0; python_version >= "3.6"
azure-functions==1.9.0
azure-storage-blob==12.10.0; python_version >= "3.6"
azure-storage-file-datalake==12.6.0; python_version >= "3.6"
bandit==1.7.4; python_version >= "3.7"
certifi==2021.10.8; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6"
cffi==1.15.0; python_version >= "3.6"
charset-normalizer==2.0.12; python_full_version >= "3.6.0" and python_version >= "3.6"
click==8.0.4; python_version >= "3.6"
colorama==0.4.4; sys_platform == "win32" and python_version >= "3.6" and python_full_version >= "3.6.2" and (python_version >= "3.7" and python_full_version < "3.0.0" and platform_system == "Windows" or platform_system == "Windows" and python_version >= "3.7" and python_full_version >= "3.5.0") and (python_version >= "3.6" and python_full_version < "3.0.0" and sys_platform == "win32" or sys_platform == "win32" and python_version >= "3.6" and python_full_version >= "3.5.0") and (python_version >= "3.6" and python_full_version < "3.0.0" and platform_system == "Windows" or platform_system == "Windows" and python_version >= "3.6" and python_full_version >= "3.5.0")
coverage==6.3.2; python_version >= "3.7"
cryptography==36.0.1; python_version >= "3.6"
dparse==0.5.1; python_version >= "3.5"
freezegun==1.2.0; python_version >= "3.6"
gitdb==4.0.9; python_version >= "3.7"
gitpython==3.1.27; python_version >= "3.7"
idna==3.3; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6"
iniconfig==1.1.1; python_version >= "3.6"
isodate==0.6.1; python_version >= "3.6"
isort==5.10.1; python_full_version >= "3.6.2" and python_version < "4.0"
lazy-object-proxy==1.7.1; python_version >= "3.6" and python_full_version >= "3.6.2"
mccabe==0.6.1; python_full_version >= "3.6.2"
msrest==0.6.21; python_version >= "3.6"
mypy-extensions==0.4.3; python_version >= "3.6"
mypy==0.931; python_version >= "3.6"
oauthlib==3.2.0; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6"
packaging==21.3; python_version >= "3.6"
pbr==5.8.1; python_version >= "3.7"
platformdirs==2.5.1; python_version >= "3.7" and python_full_version >= "3.6.2"
pluggy==1.0.0; python_version >= "3.6"
py==1.11.0; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "3.6"
pycparser==2.21; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6"
pylint==2.12.2; python_full_version >= "3.6.2"
pyparsing==3.0.7; python_version >= "3.6"
pytest-cov==3.0.0; python_version >= "3.6"
pytest==7.0.1; python_version >= "3.6"
python-dateutil==2.8.2; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.3.0" and python_version >= "3.6"
pyyaml==6.0; python_version >= "3.7"
requests-oauthlib==1.3.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6"
requests==2.27.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6"
safety==1.10.3; python_version >= "3.5"
six==1.16.0; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.3.0" and python_version >= "3.6"
smmap==5.0.0; python_version >= "3.7"
stevedore==3.5.0; python_version >= "3.7"
toml==0.10.2; python_full_version >= "3.6.2" and python_version >= "3.5"
tomli==2.0.1; python_version >= "3.7"
types-freezegun==1.1.6
typing-extensions==4.1.1; python_version < "3.10" and python_full_version >= "3.6.2" and python_version >= "3.6"
urllib3==1.26.8; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version < "4" and python_version >= "3.6"
wrapt==1.13.3; python_full_version >= "3.6.2"

**Comment by jacobtylerwalls** [comment]
Thanks for the report. Turns out this is a duplicate of PyCQA/astroid#1350. A new release of `pylint` will be out shortly with the fix.

(I'm suggesting to close because I doubt a regression test makes sense here; the changes were unit-tested in astroid, and we just did a mass update of line numbers in pylint. Let me know if folks disagree.)
